snapcraft/commands/osd.start

```diff
@@ -3,6 +3,26 @@ export SNAP_CURRENT="$(realpath "${SNAP_DATA}/..")/current"
 echo $$ > "${SNAP_CURRENT}/run/ceph-osd.pid"
 cd "${SNAP}"
 
+
+maybe_unlock() {
```

Comment:
Is `shellcheck` happy with this change?

I somehow would expect it to be complaining at least about the lack of quoting around `$dev` and `$luksname` here.

Reply:
Its not complaining as the make target actually doesn't check it :-] 
I've added the quoting you mentioned, making a note for myself to get the shell scripts checked as well